pkgs/development/tools/phpactor/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib, stdenvNoCC, fetchFromGitHub, php, composer }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "phpactor";
+  version = "2023.01.21";
+
+  src = fetchFromGitHub {
+    owner = "phpactor";
+    repo = "phpactor";
+    rev = version;
+    hash = "sha256-jWZgBEaffjQ5wCStSEe+eIi7BJt6XAQFEjmq5wvW5V8=";
+  };
+
+  buildInputs = [
+    composer
+    php
+  ];
+
+  buildPhase = ''
```

Comment:
When overriding `installPhase`, you should also provide `runHook preInstall` and `runHook postInstall`. Though, in this case, it is better to rely on the default `installPhase`. See <https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md> for more info.